### PR TITLE
Add get reversals page

### PR DESCRIPTION
**Pull request by timmy-circle**
Retrieve reversals

Page should look like 

Screen Shot 2021-01-21 at 10 25 38 PM

### CODE REVIEW

**Review by MasterXen — COMMENTED** [review]
LGTM. I assume we'll hold on merging until this is on sandbox.

File: layouts/default.vue [hunk 1]
```diff
@@ -293,6 +293,10 @@ export default class DefaultLayoutsClass extends Vue {
       title: 'GET /payments',
       to: '/debug/payments/fetch',
     },
+    {
+      title: 'GET /reversals',
```
Comment (MasterXen) [inline layouts/default.vue:297]: If this only applies to ACH, should it go lower down with the ACH-specific endpoints?

Comment (timmy-circle) [reply layouts/default.vue:297]: I thought about that. But the series of ACH endpoint below are more for creating ach account. Get reversals is more like getting payments.

Comment (timmy-circle) [reply layouts/default.vue:297]: But up to @geehsien

Comment (geehsien) [reply layouts/default.vue:297]: Can we put this second to last, after the 3 chargeback endpoints, before `GET /balances`?

Reversals IMO fit nicely next to chargebacks.

Comment (timmy-circle) [reply layouts/default.vue]: done 👍

Comment (timmy-circle) [reply layouts/default.vue]: I also create its own section in the overview as well

**Review by MasterXen — APPROVED** [review]
Verified against sandbox 👍

**Pull request merged**